docs/config/README.md

```diff
@@ -199,7 +199,7 @@ use_symbol_for_status = true
 ## Command Duration
 
 The `cmd_duration` module shows how long the last command took to execute.
-The module will be shown only if the command took longer than two seconds, or
+The module will be shown only if the command took longer than 2000 milliseconds, or
```

Comment:
At first I wasn't so sure about the utility of writing this in milliseconds, but after thinking about it, I'll agree that it's much better to retain consistency of units rather than have the intro be written with a common unit (seconds)

Reply:
How about specifying:
> 2000 milliseconds (2 seconds)

Reply:
i have amended this as suggested:)